common/app/conf/switches/FeatureSwitches.scala

```diff
@@ -15,6 +15,16 @@ trait FeatureSwitches {
     exposeClientSide = true
   )
 
+  val WebpackSwitch = Switch(
```

Comment:
Should this leave in `PerformanceSwitches` files rather than here ind `FeatureSwitches`?